### DISCUSSION TITLE
Help with architecture decision for Athenna CLI

### DISCUSSION THREAD

**Discussion by jlenon7** [AthennaIO/Cli#10] [Polls]

**Comment by jlenon7** [comment]
In @athenna/artisan is more easy to install the packages and configure the folders because the developer needs to be in the project root to run `node artisan` commands. Using the @athenna/cli we don't know exactly in what part of the project he is.

**Reply by txsoura** [reply]
Note that, if you are not in the root path, the artisan command won't work.

**Comment by txsoura** [comment]
By default, you don't need to set the install commands. The command already exists by default in all projects and is executed by the artisan runner.

**Reply by jlenon7** [reply]
Exactly!
